docs/docs/config/system.md

```diff
@@ -135,3 +135,40 @@ Following [split keyboard](../features/split-keyboards.md) settings are defined
 | `CONFIG_ZMK_SPLIT_BLE_PERIPHERAL_STACK_SIZE`            | int  | Stack size of the BLE split peripheral notify thread                       | 650                                        |
 | `CONFIG_ZMK_SPLIT_BLE_PERIPHERAL_PRIORITY`              | int  | Priority of the BLE split peripheral notify thread                         | 5                                          |
 | `CONFIG_ZMK_SPLIT_BLE_PERIPHERAL_POSITION_QUEUE_SIZE`   | int  | Max number of key state events to queue to send to the central             | 10                                         |
+
+## Snippets
+
+Snippets are a way to save common configuration separately when it applies to multiple different applications.
+
+More documentation is avaialable from [Zephyr](https://docs.zephyrproject.org/latest/build/snippets/index.html).
+
+ZMK implements the following snippets:
+
+### nrf52833-nosd
+
+On memory-constrained nRF52833 boards this snippet will extend the code partition to overwrite the Nordic SoftDevice.
+This gives 428KB for the code partition as opposed to 280KB with the Nordic SoftDevice.
+
+The added memory allows the nRF52833 to still fit displays and other memory-intensive features.
+
+Enable the snippet with`-S nrf52833-nosd` for local builds, for example:
+
+```
+west build -b nrfmicro_13_52833 -S nrf52833-nosd -- -DSHIELD=corne_left
+```
+
+Or by adding `snippet: nrf52833-nosd` to your `build.yaml` for the appropriate board:
+
+```
+  - board: nrfmicro_13_52833
+    snippet: nrf52833-nosd
+    shield: corne_left
+    cmake-args: -DCONFIG_ZMK_DISPLAY=y
+```
+
+:::warning
+Erasing the SoftDevice will not allow the board to use firmware built without the snippet.
+
+Flashing such firmware WILL brick the board.
+The only way to restore functionality after that is to re-flash the bootloader.
+:::
```

Comment:
```suggestion
:::caution
Erasing the SoftDevice will prevent the board from using firmware built without the snippet.

Flashing such firmware **will** brick the board.
The only way to restore functionality after that is to re-flash the bootloader with one that contains the appropriate SoftDevice partition.
:::
```
I think this deserves the big red box 😄

Reply:
Added a big red box and additional clarifications.